src/main/java/types/Display.java

```diff
@@ -186,4 +186,15 @@
      */
     // TODO: Make this an enum.
     String disconnectAction();
+
+    /**
+     * Delay before the graphic console disconnect action is carried out.
```

Comment:
in seconds? minutes? milliseconds?

Reply:
I Agree, time unit is important, lets add that and then we can merge

Reply:
Done.